### PR TITLE
docs: modify typos in nginx+lua doc

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:
1. modified the file name to follow naming rules
2. modified description and the first paragraph where typos occurred
3. replaced the term 'core developer' because it is inaccurate.
4. add `tags[technology]`

### CODE REVIEW

File: website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ keywords:
 - Apache APISIX
 - Lua
 - Nginx
-description: 本文由 Core developer of OpenResty、Apache APISIX Committer、深圳支流科技工程师罗泽轩撰写，介绍了 APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。
+description: 本文由深圳支流科技工程师罗泽轩撰写，介绍了 APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。罗泽轩是 OpenResty 开发者以及 Apache APISIX Committer。
```
Comment (Serendipity96) [inline website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md]: APISIX -> Apache APISIX

Comment (yzeng25) [reply website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md]: replace all APISIX with Apache APISIX

**Review by liuxiran — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

File: website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md [hunk 2]
```diff
@@ -9,33 +9,33 @@ keywords:
 - Apache APISIX
 - Lua
 - Nginx
-description: 本文由 Core developer of OpenResty、Apache APISIX Committer、深圳支流科技工程师罗泽轩撰写，介绍了 APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。
+description: 本文由深圳支流科技工程师罗泽轩撰写，介绍了 Apache APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。罗泽轩是 OpenResty 开发者以及 Apache APISIX Committer。
```
Comment (LiteSun) [inline website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md]:
```suggestion
description: 本文由深圳支流科技工程师罗泽轩撰写，介绍了 Apache APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。罗泽轩是 OpenResty 开发者以及 Apache APISIX PMC。
```

File: website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md [hunk 3]
```diff
@@ -9,33 +9,33 @@ keywords:
...
 tags: [technology]
 ---
-> [@spacewander](https://github.com/spacewander), Core developer of Apache APISIX from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+> [@spacewander](https://github.com/spacewander), Apache APISIX Committer from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
```
Comment (LiteSun) [inline website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md:15]: ditto

File: website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md [hunk 4]
```diff
@@ -9,33 +9,33 @@ keywords:
...
 >
 <!--truncate-->
 
-> 本文由 Core developer of OpenResty、Apache APISIX Committer、深圳支流科技工程师罗泽煊撰写，介绍了 APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 APISIX 带来的优势。
+> 本文由深圳支流科技工程师罗泽轩撰写，介绍了 Apache APISIX 选用 Nginx + Lua 这个技术栈的历史背景和这个技术栈为 Apache APISIX 带来的优势。罗泽轩是 OpenResty 开发者以及 Apache APISIX Committer。
```
Comment (LiteSun) [inline website/blog/2021-08-25-Why-Apache-APISIX-chose-Nginx-and-Lua.md]: ditto

**Pull request merged**